packages/angular/build/src/utils/i18n-options.ts

```diff
@@ -18,6 +18,7 @@ export interface LocaleDescription {
   translation?: Record<string, unknown>;
   dataPath?: string;
   baseHref?: string;
```

Comment:
Should be deprecate `baseHref` from the application builder?

Reply:
+1 to deprecating.